stable/openvpn/templates/NOTES.txt

```diff
@@ -18,4 +18,13 @@ POD_NAME=`kubectl get pods --namespace {{ .Release.Namespace }} -l type=openvpn
 && kubectl --namespace {{ .Release.Namespace }} exec -it $POD_NAME /etc/openvpn/setup/newClientCert.sh $KEY_NAME $SERVICE_IP \
 && kubectl --namespace {{ .Release.Namespace }} exec -it $POD_NAME cat /etc/openvpn/certs/pki/$KEY_NAME.ovpn > $KEY_NAME.ovpn
 
+Or if you are on AWS, run:
+
+POD_NAME=`kubectl get pods --namespace {{ .Release.Namespace }} -l type=openvpn | awk END'{ print $1 }'` \
+&& SERVICE_NAME=`kubectl get svc --namespace {{ .Release.Namespace }} -l type=openvpn | awk END'{ print $1 }'` \
+&& SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} $SERVICE_NAME -o jsonpath='{.status.loadBalancer.ingress[0].hostname}') \
```

Comment:
Similarly `kubectl get svc --namespace {{ .Release.Namespace }} -l type=openvpn -o jsonpath="{ .items[0].metadata.name }"`, although  you could plainly replace the `$SERVICE_NAME` with `{{ template "fullname" . }}`, f.e.  `SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].hostname}') \`